### PR TITLE
DROID-2769 App | Design | Update icons for Files

### PR DESCRIPTION
**Pull request by Nadezhda-Gurova**
---

- [ ] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: core-ui/src/main/java/com/anytypeio/anytype/core_ui/extensions/ResExtension.kt [hunk 1]
```diff
@@ -255,6 +255,8 @@ fun String?.getMimeIcon(name: String?): Int {
         MimeTypes.Category.TABLE -> R.drawable.ic_mime_table
         MimeTypes.Category.PRESENTATION -> R.drawable.ic_mime_presentation
         MimeTypes.Category.OTHER -> R.drawable.ic_mime_other
+        MimeTypes.Category.BROKEN -> R.drawable.ic_mime_broken
+        MimeTypes.Category.PHOTO -> TODO()
```
Comment (Nadezhda-Gurova) [inline core-ui/src/main/java/com/anytypeio/anytype/core_ui/extensions/ResExtension.kt:259]: I don't have information about this category.

File: core-ui/src/main/res/drawable/ic_mime_archive.xml [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
-      android:pathData="M6.5,3.0303L18.5,3.0303A3,3 0,0 1,21.5 6.0303L21.5,18.0303A3,3 0,0 1,18.5 21.0303L6.5,21.0303A3,3 0,0 1,3.5 18.0303L3.5,6.0303A3,3 0,0 1,6.5 3.0303z"
-      android:fillColor="#8C9EA5"/>
+      android:pathData="M2,1C2,0.448 2.448,0 3,0H17C17.552,0 18,0.448 18,1V19C18,19.552 17.552,20 17,20H3C2.448,20 2,19.552 2,19V1Z"
+      android:fillColor="#4F4F4F"
+      android:fillAlpha="0.08"/>
   <path
-      android:pathData="M12.4844,3H14.4844V4H12.5V5H10.5V4H12.4844V3ZM12.5,6V5H14.5V6H12.5ZM12.5,7H10.5V6H12.5V7ZM12.5,8V7H14.5V8H12.5ZM12.5,9H10.5V8H12.5V9ZM12.5,10V9H14.5V10H12.5ZM12.5,11H10.5V10H12.5V11ZM12.5,11H14.5V11.9844V12V18H10.5V11.9844H12.5V11ZM13.5,17V16H11.5V17H13.5Z"
-      android:fillColor="#ffffff"
+      android:pathData="M10,0H12V1H10V0ZM10,2H8V1H10V2ZM10,3V2H12V3H10ZM10,4H8V3H10V4ZM10,5V4H12V5H10ZM10,6H8V5H10V6ZM10,7V6H12V7H10ZM10,7V8H8V7H10ZM8,9.5C8,9.224 8.224,9 8.5,9H11.5C11.776,9 12,9.224 12,9.5V14.5C12,14.776 11.776,15 11.5,15H8.5C8.224,15 8,14.776 8,14.5V9.5ZM8.953,13H10.953V14H8.953V13Z"
+      android:fillColor="#949494"
```
Comment (konstantiniiv) [inline core-ui/src/main/res/drawable/ic_mime_archive.xml:12]: please, use color resources. here is:  color/text_secondary

File: core-ui/src/main/res/drawable/ic_mime_broken.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:pathData="M18.6,9.3C18.6,5.821 15.779,3 12.3,3C8.821,3 6,5.821 6,9.3V21L7.8,19.2L10.05,21L12.3,19.2L14.55,21L16.8,19.2L18.6,21V9.3ZM11.4,8.85C11.4,9.596 10.796,10.2 10.05,10.2C9.304,10.2 8.7,9.596 8.7,8.85C8.7,8.104 9.304,7.5 10.05,7.5C10.796,7.5 11.4,8.104 11.4,8.85ZM14.55,10.2C15.296,10.2 15.9,9.596 15.9,8.85C15.9,8.104 15.296,7.5 14.55,7.5C13.804,7.5 13.2,8.104 13.2,8.85C13.2,9.596 13.804,10.2 14.55,10.2Z"
+      android:fillColor="#DCDCDC"
```
Comment (konstantiniiv) [inline core-ui/src/main/res/drawable/ic_mime_broken.xml:8]: same

File: core-ui/src/main/res/drawable/ic_mime_image.xml [hunk 4]
```diff
@@ -1,18 +1,19 @@
...
-    android:viewportHeight="24">
+    android:width="20dp"
+    android:height="20dp"
+    android:viewportWidth="20"
+    android:viewportHeight="20">
   <path
-      android:pathData="M6.5,3L18.5,3A3,3 0,0 1,21.5 6L21.5,18A3,3 0,0 1,18.5 21L6.5,21A3,3 0,0 1,3.5 18L3.5,6A3,3 0,0 1,6.5 3z"
-      android:fillColor="#0FC8BA"/>
+      android:pathData="M18,1C18,0.448 17.552,0 17,0H3C2.448,0 2,0.448 2,1V19C2,19.552 2.448,20 3,20H17C17.552,20 18,19.552 18,19V1Z"
+      android:fillColor="#4F4F4F"
```
Comment (konstantiniiv) [inline core-ui/src/main/res/drawable/ic_mime_image.xml:8]: same

**Review by konstantiniiv — APPROVED** [review]

**Comment by Nadezhda-Gurova** [comment]
Before

beforeDayTheme
BeforeNightTheme

After 
afterNightTheme

afterDayTheme

**Review by uburoiubu — APPROVED** [review]

**Pull request merged**